src/aws/nist-800-53-rev4/rules/aws-nist-800-53-rev4-8.2.ts

```diff
@@ -92,6 +93,7 @@ export default {
       }
     }
   }`,
+  exclude: { not: { path: '@.name', equal: 'default' } },
```

Comment:
I see we are filtering in gql block and exclude block. Are both needed? How does this work?